packages/devtools_app/lib/src/memory/flutter/memory_protocol.dart

```diff
@@ -187,6 +225,12 @@ class HeapSample {
   final int external;
 
   final bool isGC;
+
+  final AdbMemoryInfo memoryInfo;
+
+  @override
+  String toString() => 'timestamp=$timestamp, rss=$rss, capacity=$capacity, '
```

Comment:
nit:  use `:` for all name-value pairs in toString for consistency with other devtools and flutter code.

Reply:
Done.